137.md

```diff
@@ -0,0 +1,35 @@
+NIP-137
+======
+
+Transport methods announcement
+-----------------------------------
+
+`draft` `optional`
+
+This NIP describes how digital services can announce how clients can reach them, it describes which network is used and depending on the network what other data is needed to reach it.
+
+## Specification
+
+A replacable event of kind `11111`, containing one or more of the following tag(s):
+
+```json
+["<network>", "<address>", "<protocol>"]
+```
+- `<network>` describes which network is used.
+- `<address>` describes where to address the service.
+- `<protocol>` describes which protocol/schema should be used
+
+## Example
+
+```json
+{
+  "kind": 11111,
+  "tags": [
+    ["clearnet", "some.domain.com", "wss"],
```

Comment:
I would emphasise on keeping domain names clear from protocol (http://) and escape characters (i.e. "/"). That way it's easier to search for domains by tags.